Cannot find module nlp

**rrule version**: 2.6.6
**OS**: OSX
**$ date**:  Sun Dec 13 19:40:34 PST 2020

## Output

Not sure why this imports are unhappy here. This is from the console in an Electron app:

```
internal/modules/cjs/loader.js:798 Uncaught (in promise) Error: Cannot find module './nlp'
Require stack:
- electron/js2c/renderer_init
    at Module._resolveFilename (internal/modules/cjs/loader.js:798)
    at Function../lib/common/reset-search-paths.ts.Module._resolveFilename (reset-search-paths.ts:40)
    at Module._load (internal/modules/cjs/loader.js:691)
    at Module._load (electron/js2c/asar.js:717)
    at Function.Module._load (electron/js2c/asar.js:717)
    at Module.require (internal/modules/cjs/loader.js:853)
    at require (internal/modules/cjs/helpers.js:74)
    at w (app.js:1)
    at s (app.js:1)
    at getnlp (eval at <anonymous> (app.js:1), <anonymous>:14790:23)
```

## Code

The error occurs when I call this (TypeScript):

```
import RRule from 'rrule';
// also tried
// import RRule from 'rrule/dist/esm/src/rrule';

...

const schedule = RRule.fromText("Every Sunday");
```

## Build

Typescript with rollup.

rollup.config.js

```
import typescript from '@rollup/plugin-typescript';
import {nodeResolve} from '@rollup/plugin-node-resolve';
import commonjs from '@rollup/plugin-commonjs';

export default {
  input: 'src/main.ts',
  output: {
    dir: '.',
    sourcemap: 'inline',
    format: 'cjs',
    exports: 'default'
  },
  external: ['obsidian'],
  plugins: [
    typescript(),
    nodeResolve({browser: true}),
    commonjs(),
  ],
};
```

--- a/src/nlp/index.ts
+++ b/src/nlp/index.ts
@@ -1,6 +1,7 @@
 import ToText, { DateFormatter, GetText } from './totext'
 import parseText from './parsetext'
 import RRule from '../index'
+import { Frequency } from '../types'
 import ENGLISH, { Language } from './i18n'
 
 /*!
@@ -108,12 +109,12 @@ const common = [
 ]
 
 ToText.IMPLEMENTED = []
-ToText.IMPLEMENTED[RRule.HOURLY] = common
-ToText.IMPLEMENTED[RRule.MINUTELY] = common
-ToText.IMPLEMENTED[RRule.DAILY] = ['byhour'].concat(common)
-ToText.IMPLEMENTED[RRule.WEEKLY] = common
-ToText.IMPLEMENTED[RRule.MONTHLY] = common
-ToText.IMPLEMENTED[RRule.YEARLY] = ['byweekno', 'byyearday'].concat(common)
+ToText.IMPLEMENTED[Frequency.HOURLY] = common
+ToText.IMPLEMENTED[Frequency.MINUTELY] = common
+ToText.IMPLEMENTED[Frequency.DAILY] = ['byhour'].concat(common)
+ToText.IMPLEMENTED[Frequency.WEEKLY] = common
+ToText.IMPLEMENTED[Frequency.MONTHLY] = common
+ToText.IMPLEMENTED[Frequency.YEARLY] = ['byweekno', 'byyearday'].concat(common)
 
 // =============================================================================
 // Export

--- a/src/rrule.ts
+++ b/src/rrule.ts
@@ -3,7 +3,7 @@ import dateutil from './dateutil'
 import IterResult, { IterArgs } from './iterresult'
 import CallbackIterResult from './callbackiterresult'
 import { Language } from './nlp/i18n'
-import { Nlp } from './nlp/index'
+import { fromText, parseText, toText, isFullyConvertible } from './nlp/index'
 import { DateFormatter, GetText } from './nlp/totext'
 import { ParsedOptions, Options, Frequency, QueryMethods, QueryMethodTypes, IterResultType } from './types'
 import { parseOptions, initializeOptions } from './parseoptions'
@@ -12,19 +12,6 @@ import { optionsToString } from './optionstostring'
 import { Cache, CacheKeys } from './cache'
 import { Weekday } from './weekday'
 import { iter } from './iter/index'
-
-interface GetNlp {
-  _nlp: Nlp
-  (): Nlp
-}
-
-const getnlp: GetNlp = function () {
-  // Lazy, runtime import to avoid circular refs.
-  if (!getnlp._nlp) {
-    getnlp._nlp = require('./nlp')
-  }
-  return getnlp._nlp
-} as GetNlp
 
 // =============================================================================
 // RRule
@@ -114,11 +101,11 @@ export default class RRule implements QueryMethods {
   }
 
   static parseText (text: string, language?: Language) {
-    return getnlp().parseText(text, language)
+    return parseText(text, language)
   }
 
   static fromText (text: string, language?: Language) {
-    return getnlp().fromText(text, language)
+    return fromText(text, language)
   }
 
   static parseString = parseString
@@ -256,11 +243,11 @@ export default class RRule implements QueryMethods {
    * to text.
    */
   toText (gettext?: GetText, language?: Language, dateFormatter?: DateFormatter) {
-    return getnlp().toText(this, gettext, language, dateFormatter)
+    return toText(this, gettext, language, dateFormatter)
   }
 
   isFullyConvertibleToText () {
-    return getnlp().isFullyConvertible(this)
+    return isFullyConvertible(this)
   }
 
   /**

